### PR TITLE
Update macos-10.13-Readme.md

### PR DESCRIPTION
**Pull request by andy-mishechkin**
Software Updates Week 4
Language and Runtime
+ Java 1.7: (Zulu 7.36.0.5-CA-macosx) (build 1.7.0_252-b10)
 + Java 1.8: (Zulu 8.44.0.9-CA-macosx) (build 1.8.0_242-b20) (default)
 + Java 11: Zulu11.37+17-CA (build 11.0.6+10-LTS)
 + Java 13: Zulu13.29+9-CA (build 13.0.2+6-MTS)
Package Management
+ Homebrew 2.2.3
Project Management
 + Welcome to Gradle 6.1
Utilities
 + Git 2.25.0
 + gpg (GnuPG) 2.2.19
Tools
+ Azure CLI 2.0.80
Browsers
 + Google Chrome 79.0.3945.130
Toolcache
Ruby 
+ 2.7.0
PyPy
+ 2.7.13
 + 3.6.9
Xamarin
Mono
+ 5.0.1.1
 + 5.2.0.224
 + 5.4.1.7
 + 5.8.1.0
 + 5.10.1.57
 + 5.12.0.309
 + 5.16.1.0
 + 5.18.1.3
 + 6.0.0.334
 + 6.4.0.208

### CODE REVIEW

File: images/macos/macos-10.13-Readme.md [hunk 1]
```diff
@@ -1,345 +1,107 @@
...
-
-- OS X 10.13.6 (17G10021) **High Sierra**
+## OS X info
+- System Version: macOS 10.13.6 (17G10021)
+- Kernel Version: Darwin 17.7.0
+- Boot Volume: Macintosh HD
+- Boot Mode: Normal
+- User Name: runner (runner)
+- Secure Virtual Memory: Enabled
+- System Integrity Protection: Enabled
```
Comment (sergey-akhalkov) [inline images/macos/macos-10.13-Readme.md]: I believe this info should not replace what we had previously.

File: images/macos/macos-10.13-Readme.md [hunk 2]
```diff
@@ -1,345 +1,107 @@
...
-- pip 19.3.1(python 3.7)
+- Pip 19.3.1 (python 2.7)
+- Pip 19.3.1 (python 3.7)
 - Miniconda 4.7.12
 
 ### Project Management
-
 - Apache Maven 3.6.3
-- Gradle 6.0.1
+- Welcome to Gradle 6.1!  - New sample projects in Gradle's documentation For more details see https://docs.gradle.org/6.1/release-notes.html Gradle 6.1
```
Comment (sergey-akhalkov) [inline images/macos/macos-10.13-Readme.md]: Please fix it.

File: images/macos/macos-10.13-Readme.md (hunk 1 above)
Comment (andy-mishechkin) [reply images/macos/macos-10.13-Readme.md]: Fixed

File: images/macos/macos-10.13-Readme.md (hunk 2 above)
Comment (andy-mishechkin) [reply images/macos/macos-10.13-Readme.md]: Done

**Comment by al-cheb** [comment]
LGTM

**Pull request merged**